### PR TITLE
Adds more details about Path Repository to the Documentation - #4412

### PR DESCRIPTION
**Pull request by potherca**
- Mentions symlinking
- Mentions absolute paths also being supported

### CODE REVIEW

**Comment by alcohol** [comment]
Formatting please

**Comment by potherca** [comment]
Explanation please.

**Comment by alcohol** [comment]
If you look at the rest of the file you might notice that we try to stay within a certain line length.

File: doc/05-repositories.md [hunk 1]
```diff
@@ -639,6 +639,10 @@ file, you can use the following configuration:
 }
 ```
 
+The local package will be symlinked if possible, in which case the output in the console will read `Symlinked from ../../packages/my-package`. If symlinking is _not_ possible the package will be copied. In that case, the console will output `Mirrored from ../../packages/my-package`.
+
+Instead of using a relative path, and absolute paths can also be used.
```
Comment (alcohol) [inline doc/05-repositories.md]: s/and/an/

**Comment by potherca** [comment]
Thanks for the review and feedback, the text has been amended.

**Comment by alcohol** [comment]
I'm not sure if putting the exact console output text in there is wise, as it might change over time. But thank you for your contribution :)

**Comment by potherca** [comment]
I was debating that with myself also. As part of the confusion surrounding this feature was in regards to when/whether symlinks or copies were used, I deemed it relevant enough to leave it in.

**Pull request merged**